.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: GitHub CI
+
+on: [push, pull_request]
+
+jobs:
+  ubuntu-build-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install deps
+      run: |
+        sudo apt-get update -y
+        sudo apt-get install -y libevent-dev libseccomp-dev git libsasl2-dev
+    - name: Build
+      run: |
+        gcc --version
+        ./autogen.sh
+        ./configure --enable-seccomp --enable-tls --enable-extstore --enable-sasl --enable-sasl-pwdb
+        make -j
+    - name: Test
+      run: make test
+    - name: Test TLS
```

Comment:
lets skip an explicit "make tls" since it will take a really long time to do both test suites. with TLS compiled "make test" will run some basic TLS tests. need to address another bug since they won't cause a failure though.